<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDPPanel</class>
 <widget class="QWidget" name="UDPPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>167</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>18</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>61</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="portEdit">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ConnectBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
